kafka/conf.yaml.example

```diff
@@ -14,12 +14,12 @@ instances:
       # newTag: test
     # user: username
     # password: password
-    # process_name_regex: .*process_name.* # Instead of specifying a host, and port. The agent can connect using the attach api.
-    #                                                                       #   This requires the JDK to be installed and the path to tools.jar to be set below.
-    # tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar # To be set when process_name_regex is set
+    # process_name_regex: .*process_name.*  # Instead of specifying a host, and port. The agent can connect using the attach api.
+    # This requires the JDK to be installed and the path to tools.jar to be set below.
```

Comment:
I actually wasn't sure if this was mis-spaced or if the line referred to an old var that should be completely removed